create-table.md

```diff
@@ -222,6 +222,28 @@ We also have other resources on indexes:
 - Create indexes for existing tables using [`CREATE INDEX`](create-index.html).
 - [Learn more about indexes](indexes.html).
 
+### Create a Table with Auto-Generated Row IDs
+
+To auto-generate unique row IDs, use an [`INT`](int.html) column with the `unique_rowid()` [function](functions-and-operators.html#id-generation-functions) as the column default:
+
+~~~ sql
+> CREATE TABLE test (
+    id INT PRIMARY KEY DEFAULT unique_rowid(), 
+    name STRING
+);
+~~~
+
+On insert, the `unique_rowid()` function will generate a default value from the insert timestamp and the ID of the node executing the insert, a combination that is guaranteed to be globally unique. 
```

Comment:
I'd mention `BYTES DEFAULT uuid_v4()` here too. It provides stronger guarantees of uniqueness (I'd only use the word "guarantee" with `uuid_v4`), and a different distribution (uniform instead of time-based). Some applications will prefer one distribution and others will prefer the other.